### DISCUSSION TITLE
Help with BOSCH smart plug BSP-FZ2 (RBSH-SP-ZB-EU)

### DISCUSSION THREAD

**Discussion by joachim-beck** [Koenkk/zigbee2mqtt#16625] [Devices]
Hello,
i tried to pair a Bosch BSP-FZ2 smart plug. It should be supported, but after pairing with the install code, i get always the following message:

2023-02-11 19:02:31Received message from unsupported device with Zigbee model 'RBSH-SP-ZB-EU' and manufacturer name 'BOSCH'
2023-02-11 19:02:31Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

What i am doing wrong here?
I am using the latest version of Zugbee2Mqtt from the dev channel.

Thank you for some help.
Joachim

**Comment by Koenkk** [comment]
`RBSH-SP-ZB-EU` should be supported in z2m 1.30.1, so you are not using the latest dev version (to update the edge addon you need to uninstall/install it)

**Comment by joachim-beck** [comment]
Hello,
now i am a little bit irritated. If I look at the web interface of zigbee2mqtt i see the version "1.30.1-dev commit: e9a38353".
Is that not the right one? (see attached screenshot)
Thank you
Joachim

![image](https://user-images.githubusercontent.com/117013215/218320755-e1dc42a0-477a-483e-a5e4-4014a8e0cb99.png)

**Reply by Koenkk** [reply]
That should be the right one, how are you running z2m?

**Comment by joachim-beck** [comment]
Hello,
Zigbee2MQTT is running under Microsoft Server. In the meantime I reinstalled Zigbee2MQTT again, and now, I don’t know why, it is working.
Thank you for your support.
Regards 
Joachim
